packages/components/tooltip/src/tooltip.tsx

```diff
@@ -181,7 +189,7 @@ export const Tooltip = motionForwardRef<TooltipProps, "div">(
       "Tooltip",
       props,
     )
-    const {
+    let {
```

Comment:
Why did you change from const to let? Is there a reason for that?

Reply:
Oops, sorry, I was testing around locally and forgot to change it back, I have changed it back :)